ddp-workspace/projects/ddp-dsm-ui/src/app/STORE/actions/participants.actions.ts

```diff
@@ -0,0 +1,33 @@
+import {createAction} from '@ngrx/store';
+import {getParticipant, getParticipants} from './enums/participants-enums';
+import {ViewFilter} from '../../filter-column/models/view-filter.model';
+import {Sort} from '../../sort/sort.model';
+import {ParticipantModel} from '../models/participant.model';
+
+export const getParticipantsRequest = createAction(
+  getParticipants.REQUEST,
+  (viewFilter: ViewFilter, realm: string, parent: string, filterQuery: string,
+   from: number, to: number, sort: Sort) => ({viewFilter, realm, parent, filterQuery, from, to, sort})
+);
```

Comment:
We have 3 different states for actions that also are triggering HTTP requests: Request, Success, and Failure, which is helping us to have control over all expected statuses of specific requests.